Reaktoro/Core/Phases.cpp

```diff
@@ -1,350 +1,367 @@
-// Reaktoro is a unified framework for modeling chemically reactive systems.

```

Comment:
I don't understand why this file is marked here as completely different. It's not possible to see the minor changes that occurred in it.

Reply:
The same.